Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -117,5 +148,21 @@ public void OnInputClicked(InputEventData eventData)
                 anchorManager.AttachAnchor(gameObject, SavedAnchorFriendlyName);
             }
         }
+
+        private void DetermineParent()
+        {
+            if (ParentGameObjectToPlace == null)
+            {
+                if (gameObject.transform.parent == null)
+                {
+                    Debug.LogError("The selected GameObject has no parent.");
+                    PlaceParentOnTap = false;
+                }
+                else
+                {
+                    ParentGameObjectToPlace = gameObject.transform.parent.gameObject;
```

Comment:
It might be good to print out the name of the parent auto-selected here, just in case the desire was actually to use a GameObject higher in the hierarchy but it wasn't set properly.